src/reducers/auth.js

```diff
@@ -1,7 +1,7 @@
 import * as types from '../actions/types';
 
 const INITIAL_STATE = {
-  authenticated: false,
+  authenticated: true,
```

Comment:
```suggestion
  authenticated: false,
```
Reset this back to false